CHANGELOG.md

```diff
@@ -12,6 +12,14 @@
   module's directory.
 - Fix constraints of `--path` flag for lint and breaking rules to avoid resolving all files
   within a module. This change can result in a performance improvement for large workspaces.
+- Add `includes` key to module configurations in v2 `buf.yaml`, accepting a list of directories.
+  * If `includes` is specified, a proto file is considered in the module only if it is in one of the
+    directories specified.
+  * If both `includes` and `excludes` keys are specified for a module, a proto file is considered
+    part of this module if it is contained in any of the include paths and not in any of the exclude
+    paths.
+  * This key does not affect license files or documentation files, e.g. `README.md`.
```

Comment:
Is this also the case with excludes?

Reply:
I don't think it's applicable with `excludes`. If a module is at `proto`, its exclude paths must be in the form of `proto/some_dir` and cannot exclude `proto/LICENSE`.

Reply:
OK so are both includes and excludes directories only, or do they also allow specifying `.proto` files?

Reply:
They are both directories only.

Reply:
I don't understand the changelog then - why are license and doc files being referenced?

Reply:
I thought specifying `moduleDirPath/subdir` as an `includes:` path might give the impression that `moduleDirPath/LICENSE` is left out (which is not the case with `excludes:`).

Reply:
This should be removed - it confuses me that we're referencing license and readme at all here.